pages/aboutus.js

```diff
@@ -3,24 +3,77 @@ import Footer from '../components/footer'
 import Head from '../components/head'
 import '../static/styles/main.scss'
 
+const employees = [
```

Comment:
Can we put this info in the API?

Reply:
yea i'll add a todo, will keep static for now for styling purposes